commands/dev-deploy.js

```diff
@@ -74,8 +74,15 @@ async function createDevAccountIfNeeded({ near, keyStore, networkId, init }) {
             }
         }
     }
+    let accountId;
+    if (typeof masterAccount === 'undefined' && helperUrl) {
+        // determine "faucet" account name of environment from helperUrl
+        const splitHelper = helperUrl.split('.');
+        accountId = `dev-${Date.now()}.${splitHelper[1]}`;
```

Comment:
this seems to be inconsistent with `tla` logic added in other part.

I feel it will all be much easier if we just always use `dev-${Date.now()}.${masterAccount}`;

Reply:
So `dev-deploy` can use *either* `--helperUrl` which is what it'll be 99% of the time, **or** you can use `--masterAccount`
Am I misunderstanding `dev-deploy` or are you a bit confused. I know this PR has been open for a while and my brain is straining at this point, too.

Reply:
I am, however, adding `tla` instead of parsing the `helperUrl`. I definitely missed updating this file after adding that new key